ibis/backends/sql/rewrites.py

```diff
@@ -351,6 +408,46 @@ def wrap(node, _, **kwargs):
 # supplemental rewrites selectively used on a per-backend basis
 
 
+@replace(Select)
+def split_select_distinct_with_order_by(_):
```

Comment:
There are dragons here.

Reply:
Missed opportunity to call this `supplant_merged_with_ordered_distinct` 😂